.gitignore

```diff
@@ -122,3 +122,10 @@ env/
 
 ### Vagrant ###
 .vagrant
+
+### Terraform ###
+.terraform
+.terraform.lock.hcl
```

Comment:
It's worth noting that this file is a little like a package-lock.json or poetry.lock, and thus should be checked in [as per the terraform docs](https://www.terraform.io/docs/language/dependency-lock.html#lock-file-location) to your git repo

Reply:
removed .terraform.lock.hcl from .gitignore